packages/react-composites/src/composites/CallWithChatComposite/CallWithChatPane.tsx

```diff
@@ -102,8 +103,8 @@ export const CallWithChatPane = (props: {
       <Stack.Item verticalFill grow styles={paneBodyContainer}>
         <Stack horizontal styles={scrollableContainer}>
           <Stack.Item verticalFill styles={scrollableContainerContents}>
-            <Stack styles={props.activePane === 'chat' ? availableSpaceStyles : hiddenStyles}>{chatContent}</Stack>
-            <Stack styles={props.activePane === 'people' ? availableSpaceStyles : hiddenStyles}>{peopleContent}</Stack>
+            {props.activePane === 'chat' && <Stack styles={availableSpaceStyles}>{chatContent}</Stack>}
```

Comment:
The comment got lost through refactors but we previously hide the chat pane instead of not rendering it as an easy fix to prevent partially composed chat messages being lost (https://github.com/Azure/communication-ui-library/pull/662) -- we'll need to either find a different fix for maintaining partially composed messages or find a different way to trigger the autofocus.

I worry about not rendering the chat pane as quite a large change that could have potential regressions before right release.

I think this exposes that we really should have a function in the adapter that can set focus but the API design was tough so we settled on just the autofocus prop /:

Reply:
Ahh I see, I did think it was interesting that it was just hidden and not rendered, so is the issue in that if it is de-rendered when messages come in it can lose some of the messages in the thread because the composite isn't present, and when it shows up it only grabs some of the messages in the thread?

Reply:
Not that messages in the message thread is lost -- its that anything typed into the sendbox _but not sent yet_ is lost.
i.e. open chat pane > type a message (do not hit send) > close chat pane > reopen chat pane > typed message is gone when it shouldn't be